src/utils/getNewId.ts

```diff
@@ -0,0 +1,7 @@
+import { Todo } from '../types';
+
+export const getNewId = (todos: Todo[]) => {
+  const sortedTodos = todos.sort((todo1, todo2) => todo2.id - todo1.id);
```

Comment:
The `sort` method mutates the original array, which can lead to unexpected behavior if `todos` is used elsewhere in the application. Consider using `slice()` to create a shallow copy of the array before sorting to avoid side effects: `const sortedTodos = todos.slice().sort((todo1, todo2) => todo2.id - todo1.id);`